src/repos.js

```diff
@@ -64,42 +44,17 @@ export const repos = {
       distribution_arch: 'x86_64',
       url: 'http://mirror.centos.org/centos/8-stream/extras/x86_64/os/',
     },
-    {
-      name: 'google-compute-engine',
-      distribution_arch: 'x86_64',
-      url: 'https://packages.cloud.google.com/yum/repos/google-compute-engine-el8-x86_64-stable',
-    },
-    {
-      name: 'google-cloud-sdk',
-      distribution_arch: 'x86_64',
-      url: 'https://packages.cloud.google.com/yum/repos/cloud-sdk-el8-x86_64',
-    },
   ],
   'centos-9': [
     {
       name: 'baseos',
       distribution_arch: 'x86_64',
-      url: 'http://mirror.centos.org/centos/9-stream/BaseOS/x86_64/os/',
+      url: 'https://mirror.stream.centos.org/9-stream/BaseOS/x86_64/os/',
     },
     {
       name: 'appstream',
       distribution_arch: 'x86_64',
-      url: 'http://mirror.centos.org/centos/9-stream/AppStream/x86_64/os/',
-    },
-    {
-      name: 'extras',
-      distribution_arch: 'x86_64',
-      url: 'http://mirror.centos.org/centos/9-stream/extras/x86_64/os/',
-    },
-    {
-      name: 'google-compute-engine',
-      distribution_arch: 'x86_64',
-      url: 'https://packages.cloud.google.com/yum/repos/google-compute-engine-el9-x86_64-stable',
-    },
-    {
-      name: 'google-cloud-sdk',
-      distribution_arch: 'x86_64',
-      url: 'https://packages.cloud.google.com/yum/repos/cloud-sdk-el9-x86_64',
+      url: 'https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/',
```

Comment:
I'm getting "No packages found" with `https`, but with `http` it seems to work

Reply:
ugh, pushed local changes when fixing the linter error :facepalm:

Reply:
Very odd! Accessing the URL directly works for me.

Reply:
It's more that the content service saved it as http, and doesn't recognize the https url => no packages if the https url is used.

Reply:
Ah!

Reply:
We probably should avoid non https by default, no? At least if https is available. Is Justin aware of this quirk?